test/com/sun/jna/NativeLibraryTest.java

```diff
@@ -255,18 +255,19 @@ public void testMatchFramework() {
         if (!Platform.isMac()) {
             return;
         }
-        final String[][] MAPPINGS = {
-            // Depending on the system, /Library/Frameworks may or may not
-            // have anything in it.
-            { "QtCore", expected("/Library/Frameworks/QtCore.framework/QtCore") },
-            { "Adobe AIR", expected("/Library/Frameworks/Adobe AIR.framework/Adobe AIR") },
-
-            { "QuickTime", expected("/System/Library/Frameworks/QuickTime.framework/QuickTime") },
-            { "QuickTime.framework/Versions/Current/QuickTime", expected("/System/Library/Frameworks/QuickTime.framework/Versions/Current/QuickTime") },
-        };
-        for (int i=0;i < MAPPINGS.length;i++) {
-            assertEquals("Wrong framework mapping", MAPPINGS[i][1], NativeLibrary.matchFramework(MAPPINGS[i][0]));
+        if (!System.getProperty("os.version").startsWith("10")) {
```

Comment:
Note: `System.getProperty("os.version")` currently returns `10.16` for developer beta 1. My understanding is this will eventually become `11.00` (or possibly `11.0`?) in a subsequent build, and may already be reporting that on the Apple DTK ARM build. But for the time being this check on just `startsWith("10")` won't work on DP1

Reply:
`System.getProperty("os.version").compareTo("10.15") > 0` should work.